chain/vm/runtime.go

```diff
@@ -341,7 +341,7 @@ func (rt *Runtime) Send(to address.Address, method abi.MethodNum, m vmr.CBORMars
 	if m != nil {
 		buf := new(bytes.Buffer)
 		if err := m.MarshalCBOR(buf); err != nil {
-			rt.Abortf(exitcode.SysErrInvalidParameters, "failed to marshal input parameters: %s", err)
+			rt.Abortf(exitcode.ErrSerialization, "failed to marshal input parameters: %s", err)
```

Comment:
This is a consensus-affecting change. It should be behind a network version check.

Reply:
@anorth This one "sneaked" in -- it doesn't break consensus pre-upgrade, (and won't break things moving forward).

Reply:
How does it not break consensus? It'll change the error codes (or do those not go on chain?).